server/src/test/java/org/elasticsearch/index/shard/RefreshListenersTests.java

```diff
@@ -178,7 +178,7 @@ public void testBeforeRefresh() throws Exception {
         assertEquals(1, listeners.pendingCount());
 
         TestSeqNoListener seqNoListener = new TestSeqNoListener();
-        assertFalse(listeners.addOrNotify(index.getSeqNo(), seqNoListener));
+        assertFalse(listeners.addOrNotify(index.getSeqNo(), false, seqNoListener));
```

Comment:
Also randomBoolean here? As well as in other changes where `false` was added in this class except inside `testSequenceNumberMustBeIssued`.

```suggestion
        assertFalse(listeners.addOrNotify(index.getSeqNo(), randomBoolean(), seqNoListener));
```